AISKU/Initialization.ts

```diff
@@ -91,18 +92,16 @@ export class Initialization implements IApplicationInsights {
         this.core = new AppInsightsCore();
         let extensions = [];
         let appInsightsChannel: Sender = new Sender();
+        const windowName: string = this.config.extensionConfig.ApplicationInsightsAnalytics.sdkInstanceName || Initialization.defaultName;
 
         extensions.push(appInsightsChannel);
         extensions.push(this.properties);
         extensions.push(this.dependencies);
         extensions.push(this.appInsights);
 
         // initialize core
+        window[windowName] = this;
```

Comment:
you should not need to do this at this point. This should have been setup through loadAppInsights() return call.